autojustice/justicetools.lua

```diff
@@ -0,0 +1,176 @@
+--@ module = true
+
```

Comment:
if you want to keep this separate, this file should be moved to `internal/autojustice/justicetools.lua` so it doesn't show up as a runnable script. However, together they are about 500 lines, which is about average for a script like this. It might be simpler if you just combine the two.

Reply:
I also think that it's simpler to combine the two. My only reasoning to keep these files separate is just to facilitate in case others might want get/change some info from the justice system.